db/event_handler.go

```diff
@@ -132,10 +133,8 @@ func (wh *Webhook) HandleEvent(event Event) {
 			return
 		}
 
-		if base.LogEnabled("Events+") {
-			base.Debugf(base.KeyEvents, "Webhook handler ran for event.  Payload %s posted to URL %s, got status %s",
-				base.UD(payload), base.UD(wh.SanitizedUrl()), resp.Status)
-		}
+		base.Debugf(base.KeyEvents, "Webhook handler ran for event.  Payload %s posted to URL %s, got status %s",
```

Comment:
I think we still want the LogEnabled check to avoid the wh.SanitizedUrl() overhead.  We want to change this to `if base.DebugEnabled()`.